packages/@okta/i18n/src/properties/login.properties

```diff
@@ -915,6 +915,9 @@ oie.phone.invalid = Invalid Phone Number.
 oie.email.label = Email
 oie.email.authenticator.description = Verify with a link or code sent to your email
 oie.email.mfa.title = Verify with your email
+oie.email.verify.primaryButton = Send me an email
+# {0} is a user's email address
+oie.email.verify.subtitle = Verify with an email link or enter a code sent to <span class="strong no-translate">{0}</span>
```

Comment:
let's change this so we don't use html tags in here. Let me find the wiki about this

Reply:
good catch @mauriciocastillosilva-okta